www/js/join/join-ctrl.js

```diff
@@ -33,6 +35,13 @@ angular.module('emission.join.ctrl', ['emission.splash.startprefs',
       $scope.popover.hide($event)
     }
 
+    function handleOpenURL(url) {
+      console.log("onLaunch method from external function called");
+      var c = document.querySelectorAll("[ng-app]")[0];
+      var scope = angular.element(c).scope();
+      scope.$broadcast("CUSTOM_URL_LAUNCH", url);
+    };
+
```

Comment:
I remember copying this. I don't know why; I was focused on something else
TODO: check it out

Reply:
Before being moved, this function was defined at the root level `www/js/splash/customURL.js`. It was not exported.
So before the project used `webpack` and ES modules, this function lived on the global scope.

Yet, it is only ever used here in `join-ctrl.js`. So I think moving it here is fine for now.

If it is important for it to be moved back to `customURL.js`, it would need to be a named export.